python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -811,10 +813,14 @@ def _actually_load_xml(self, fobj, fn, this_id, load_backup):
             a4=time.time()
             logger.debug("Loading XML file for ID: %s took %s sec" % (this_id, str(a4-b4)))
 
+            if errs is not None:
+                if len(errs) > 0:
```

Comment:
Unless we're expecting a `None` I see no reason to test for it. The variable is a `list` and we should be able to rely on that.